packages/hello-gsm-admin/src/components/Modal/ModalSubmit/index.tsx

```diff
@@ -5,7 +5,11 @@ import * as S from './style';
 import useStore from 'Stores/StoreContainer';
 import { CommonApplicationResponseType } from 'type/application';
 
-const ModalSubmit = ({ data }: { data: CommonApplicationResponseType }) => {
+interface Props {
+  data: CommonApplicationResponseType;
+}
```

Comment:
컴포넌트명 + Props로 지정하는게 더 명확할 것 같습니다!

```suggestion
interface ModalSubmitProps {
  data: CommonApplicationResponseType;
}
```

Reply:
아 넵 감사합니다!